Sources/Purchasing/OfferingsManager.swift

```diff
@@ -134,9 +144,19 @@ private extension OfferingsManager {
             let missingProductIDs = self.getMissingProductIDs(productIDsFromStore: Set(productsByID.keys),
                                                               productIDsFromBackend: productIdentifiers)
             if !missingProductIDs.isEmpty {
-                Logger.appleWarning(
-                    Strings.offering.cannot_find_product_configuration_error(identifiers: missingProductIDs)
-                )
+                switch fetchPolicy {
+                case .ignoreNotFoundProducts:
+                    Logger.appleWarning(
```

Comment:
Do we need to `return` after this?

Reply:
No, not in this case. This is the existing behavior, which was to simply log and keep going with the products that are found.